web/impact/impact/v1/classes/application_data_cache.py

```diff
@@ -27,6 +28,10 @@ def __init__(self, apps, criteria, feedback):
                 "fields": field_data,
             }
 
+    def _helpers(self):
```

Comment:
I think this should create a dictionary instead of list, It will be useful in instances such as [this](https://github.com/masschallenge/impact-api/pull/250/files#diff-46ea2a56bf6829c715519480ec9f7579R19) as well as everywhere else its being used currently with a `.values()` on the dict